webapp/src/components/sidebar/sidebarCategory.tsx

```diff
@@ -248,10 +299,25 @@ const SidebarCategory = (props: Props) => {
                     />
                 )
             })}
-            {!collapsed && props.boards.map((board: Board) => {
+            {!collapsed && props.boards.map((board: Board, index) => {
                 if (!isBoardVisible(board.id)) {
                     return null
                 }
+                if(index === 0) {
+                    return (
+                        <SidebarBoardItem
```

Comment:
The only difference betwene the if and else block is with `shouldViewManageBoardsTour` prop being passed in one of those. We can avoid the if else and just have one SidebarBoardItem with `shouldViewManageBoardsTour={index === 0 ? shouldViewManageBoardsTour : undefined}`

Reply:
Correct will update this.